packages/app/src/update/requests/get-all-releases.request.ts

```diff
@@ -22,11 +20,11 @@ const productsMapper = {
   [DeviceType.MuditaHarmony]: Product.BellHybrid,
 }
 
-const getAllReleases = async (
+export const getAllReleases = async (
   deviceType: DeviceType
 ): Promise<AllReleasesResponse> => {
   const releases: Release[] = await ipcRenderer.callMain<undefined, Release[]>(
-    GetAllReleasesEvents.Request
+    IpcUpdate.GetDevelopmentRelease
```

Comment:
the same case like in the https://github.com/mudita/mudita-center/pull/629#discussion_r802593895

Reply:
https://github.com/mudita/mudita-center/pull/629/commits/8116b41233329e3201474e7769aaab262b95ca20